### DISCUSSION TITLE
Welcome to VanillaTweaksBedrock Discussions!

### DISCUSSION THREAD

**Discussion by PiSaucer** [PiSaucer/VanillaTweaksBedrock#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by edisonadam** [comment]
You can add a way to download the pack with everything but individually.

**Reply by PiSaucer** [reply]
What? Like just all the files that are already on the GitHub repo page? Here is the [Main.zip](https://github.com/PiSaucer/VanillaTweaksBedrock/archive/refs/heads/main.zip) of the entire repo.
